app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,22 @@
+.main-section {
+  align-items: center;
+  border-top: 15px solid $orange;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: 20px;
+  width: auto;
```

Comment:
this is the default value, if you set the body to use flex and add direction column it'll be easier to implement the rest